### PR TITLE
Modify init

### PR DESCRIPTION
**Pull request by r0mainK**
Do not delete the bucket if it exists

### CODE REVIEW

File: modelforge/gcs_backend.py [hunk 1]
```diff
@@ -100,8 +100,10 @@ def reset(self, force):
                 self._log.error("Bucket already exists, aborting.")
                 raise ExistingBackendError
             self._log.info("Bucket already exists, deleting all content.")
-            bucket.delete(force, client)
-        client.create_bucket(self.bucket_name)
+            for blob in bucket.list_blobs():
```
Comment (vmarkovtsev) [inline modelforge/gcs_backend.py]: Should be an "info" log message for each blob removal

Comment (r0mainK) [reply modelforge/gcs_backend.py]: done

Comment (vmarkovtsev) [reply modelforge/gcs_backend.py]: Are you sure `% blob` will print what it should? Is it `blob.name` or smth else?

Comment (r0mainK) [reply modelforge/gcs_backend.py]: yes my bad it was in fact blob.name, i thought `blob` was an url but it's an object

**Pull request merged**